src/main/java/com/denizenscript/depenizen/bukkit/commands/worldedit/WorldEditCommand.java

```diff
@@ -0,0 +1,363 @@
+    // Use to paste a schematic at a location with a player attached to the edit history.
+    // - worldedit schematicpaste file:<[filepath]> position:<player.location> undoable noair rotate:90
+    //
+    // -->
+
+    private enum Action {CUBOIDSCHEMATIC, CUBOIDCLIPBOARD, SCHEMATICPASTE, SCHEMATICCLIPBOARD}
+
+    @Override
+    public void parseArgs(ScriptEntry scriptEntry) throws InvalidArgumentsException {
+
+        for (Argument arg : scriptEntry.getProcessedArgs()) {
+
+            if (!scriptEntry.hasObject("position")
+                    && arg.matchesPrefix("position")) {
+                scriptEntry.addObject("position", arg.asType(LocationTag.class));
+            }
+
+            else if (!scriptEntry.hasObject("file")
+                    && arg.matchesPrefix("file")) {
+                scriptEntry.addObject("file", arg.asElement());
+            }
+
+            else if (!scriptEntry.hasObject("cuboid")
+                    && arg.matchesPrefix("cuboid")) {
+                scriptEntry.addObject("cuboid", arg.asType(CuboidTag.class));
+            }
+
+            else if (!scriptEntry.hasObject("noair")
+                    && arg.matches("noair")) {
+                scriptEntry.addObject("noair", new ElementTag(true));
+            }
+
+            else if (!scriptEntry.hasObject("undoable")
+                    && arg.matches("undoable")) {
+                scriptEntry.addObject("undoable", new ElementTag(true));
+            }
+
+            else if (!scriptEntry.hasObject("rotate")
+                    && arg.matchesPrefix("rotate")) {
+                scriptEntry.addObject("rotate", arg.asElement());
+            }
+
+            else if (!scriptEntry.hasObject("action")
+                    && arg.matchesEnum(Action.values())) {
+                scriptEntry.addObject("action", arg.asElement());
+            }
+
+            else {
+                arg.reportUnhandled();
+            }
+
+        }
+        if (!scriptEntry.hasObject("action")) {
+            throw new InvalidArgumentsException("Action not specified!");
+        }
+        if (Utilities.entryHasPlayer(scriptEntry)) {
+            scriptEntry.addObject("target", Utilities.getEntryPlayer(scriptEntry));
+        }
+    }
+
+    @SuppressWarnings("unchecked")
+    @Override
+    public void execute(ScriptEntry scriptEntry) {
+
+        ElementTag file = scriptEntry.getObjectTag("file");
+        ElementTag action = scriptEntry.getObjectTag("action");
+        PlayerTag target = scriptEntry.getObjectTag("target");
+        LocationTag position = scriptEntry.getObjectTag("position");
+        ElementTag noAir = scriptEntry.getObjectTag("noair");
+        CuboidTag cuboid = scriptEntry.getObjectTag("cuboid");
+        ElementTag id = scriptEntry.getObjectTag("id");
+        ElementTag data = scriptEntry.getObjectTag("data");
+        ElementTag undoable = scriptEntry.getObjectTag("undoable");
+        ElementTag rotate = scriptEntry.getObjectTag("rotate");
```

Comment:
Several of the options above (id, data, ...) don't correspond to anything that's ever set in parseArgs.